authoringAssistant/testCases/scigen-multiple-replace/1904.12550v1-6.json

```diff
@@ -0,0 +1,19 @@
+{
+  "datasets": [
+    {
+      "var": "tableData",
+      "file": "datasets/scigen/1904.12550v1-6"
+    }
+  ],
+  "imports": [],
+  "variables": {},
+  "expected": {
+    "num_1": "head (map (fun x -> x.f_818) (filter (fun x -> (x.gong_et_al == \"AVG_COS_SIM\") `and` (x.topic_science == \"Both\")) tableData))"
+  },
+  "paragraph": [
+    {
+      "type": "literal",
+      "value": "The results can be found in Table 3. For comparison, the two top rows provide the best results of Gong et al. (2018). The first interesting finding is that the AVG COS SIM measure again performs very well: In all three settings, it beats both the system based on general-purpose embeddings (topic wiki) and the one that is adapted to the science domain (topic science), with again the Both setting yielding the best overall result ([REPLACE id=\"num_1\" value=\"0.926\"]). Note that our Both setting is probably the one most similar to the concept input used by Gong et al. (2018)."
```

Comment:
It might be nice to include the two occurrences of `Gong et al. (2018)` as computed values too